package.json

```diff
@@ -18,9 +19,11 @@
   },
   "homepage": "https://github.com/pelias/query#readme",
   "dependencies": {
-    "check-types": "^7.0.0"
+    "check-types": "^7.0.0",
+    "jshint": "^2.9.2"
```

Comment:
should this be in both dependencies and devDependencies and with different versions?